doc/website/advanced/configuration-guide.md

```diff
@@ -48,11 +68,14 @@ The value for the alignment is set to 8.
 
 ### :material-file-cog: Dynamic configuration
 
-One way is to read a configuration dynamically at RouDi runtime (startup).
-Using TOML Config in RouDi is not mandatory for configuring segments and mempools, but a comfortable alternative.
+One way is to read a configuration dynamically during the startup of RouDi.
+Using the TOML Config in RouDi is not mandatory for configuring segments and
+mempools, but a comfortable alternative.
 
-To enable the TOML config in iceoryx, the CMake option `-DTOML_CONFIG=ON` must be used (enabled by default).
-The `iox-roudi` provided by iceoryx is with TOML support and can be used out of the box.
+To enable the TOML config in iceoryx, the CMake option `-DTOML_CONFIG=ON` must
+be enabled (this is the default setting).
```

Comment:
This reads somehow weird. If it is the default setting why writing about enabling it. What do you think of
```
The CMake option `-DTOML_CONFIG`, which is `ON` by default, enables the TOML config for iceoryx.
```